recipe/build.sh

```diff
@@ -3,13 +3,35 @@ set -exv
 # This step is required when building from raw source archive
 make generate --jobs ${CPU_COUNT}
 
+# Only about 7 virtual archs can be built 6 hours for CUDA 11
+
 # Duplicate lists because of https://bitbucket.org/icl/magma/pull-requests/32
-export CUDA_ARCH_LIST="sm_35,sm_50,sm_60,sm_70,sm_80"
-export CUDAARCHS="35-virtual;50-virtual;60-virtual;70-virtual;80-virtual"
+export CUDA_ARCH_LIST="sm_35,sm_50,sm_60,sm_61,sm_70,sm_75,sm_80"
+export CUDAARCHS="35-virtual;50-virtual;60-virtual;61-virtual;70-virtual;75-virtual;80-virtual"
+
+if [[ "$cuda_compiler_version" == "12.0" ]]; then
```

Comment:
Would suggest making CUDA 11 the special case. The advantage being that when we drop it, we can just delete som lines from the recipe. Also it will make adding new CUDA 12.x's easier